TM1py/Services/CellService.py

```diff
@@ -156,6 +157,16 @@ def clear_spread(
 
         return self._post_against_cellset(cellset_id=cellset_id, payload=payload, delete_cellset=True, **kwargs)
 
+    def clear_with_mdx(self, cube: str, mdx: str, **kwargs) -> Response:
+        view_name = "".join(['}TM1py', str(uuid.uuid4())])
+        ti = [
+            f"ViewCreateByMdx('{cube}','{view_name}','{mdx}',1);",
+            f"ViewZeroOut('{cube}','{view_name}');"]
+
+        from TM1py import ProcessService
+        process_service = ProcessService(self._rest)
+        return process_service.execute_ti_code(lines_prolog=ti, **kwargs)
```

Comment:
I suggest using execute_process_with_return instead. It eliminates dependencies on uuid and removes the burden of creating and deleting a TI on the server, which sometimes can be prone to lock contention.

Reply:
Good point! Thanks for the catch.